pkg/devices/app.jsx

```diff
@@ -0,0 +1,68 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+
+import { Listing, ListingRow } from 'cockpit-components-listing.jsx';
+import PCIDevices, { pciActions } from './pci.jsx';
+
+const Router = ({ deviceActions, state }) => {
+    const { visibility, pciDevices, pciDrivers } = state;
+
+    /* TODO: Add USB and SCSCI devices */
+    switch (visibility.bus) {
+        case 'pci':
+        default:
+            return (<PCIDevices visibility={visibility} pciDevices={pciDevices} pciDrivers={pciDrivers} deviceActions={deviceActions} />)
+    }
+};
+
+const NavBar = ({ deviceActions, visibility }) => {
+    const active = visibility.bus;
+    const isActive = (bus) => bus === active ? 'active' : '';
+
+    return (
+        <div className='content-header-extra'>
+            <div className='btn-group'>
+                <button className={`btn btn-default ${isActive('pci')}`} onClick={() => deviceActions.onPciClassSelected(null)}>PCI</button>
+                <button className={`btn btn-default ${isActive('scsi')}`}>SCSI</button>
+                <button className={`btn btn-default ${isActive('usb')}`}>USB</button>
```

Comment:
I'll mark each block where I find untranslated strings. At the top of the file you can have:

```
const _ = cockpit.gettext
```

or equivalent.